success/index.js

```diff
@@ -1,66 +1,213 @@
-(() => {
-  // Parse query string from URL
-  const qs = ((a) => {
-    if (a === '') return {};
-    var b = {}
-    for (var i = 0; i < a.length; ++i) {
-      var p = a[i].split('=')
-      if (p.length !== 2) continue
-      b[p[0]] = decodeURIComponent(p[1].replace(/\+/g, ' '))
+(async () => {
+  /* Fetch and Parse Location */
+  const hpAdminUrl = (hpAdminQueryString => {
```

Comment:
I think this would be simpler without wrapping it in a function call. Just  add `const hpAdminQueryString = window.location.search.substr(1).split('&')`

Reply:
`window.location.searchParams`, rather, which is already an object.